_pytest/doctest.py

```diff
@@ -291,6 +295,18 @@ def _get_allow_bytes_flag():
     return doctest.register_optionflag('ALLOW_BYTES')
 
 
+def _get_report_choices():
+    import doctest
+    return dict(
+        zip(
+            _get_report_choices_keys(),
+            (doctest.REPORT_UDIFF, doctest.REPORT_CDIFF, doctest.REPORT_NDIFF, doctest.REPORT_ONLY_FIRST_FAILURE, 0, )
+        )
+    )
+
+def _get_report_choices_keys():
```

Comment:
It was the first implementation, but then it required to import doctest at the option parsing time, and it was making test suite fail (because i guess some doctest dependency is importing logging, and there's a test that checks it is not the case.)

See discussion at #1749 (and first impl at 625b603f1f414e59f306116ba8194673fdb0ce83 that broke the tests). Let me know if you want me to change back but then I have no idea how to avoid this import related failure.